validation/src/test/scala/hmda/validation/rules/lar/validity/V540Spec.scala

```diff
@@ -7,30 +7,31 @@ import org.scalacheck.Gen
 
 class V540Spec extends LarEditCheckSpec with BadValueUtils {
 
-  val badActionTakenList = List(2, 3, 4, 5, 7, 8)
-  val validActionTakenGen: Gen[Int] =
+  val relevantActionTakenList = List(2, 3, 4, 5, 7, 8)
+
+  val irrelevantActionTakenGen: Gen[Int] =
     Gen.choose(Int.MinValue, Int.MaxValue)
-      .filter(!badActionTakenList.contains(_))
+      .filter(!relevantActionTakenList.contains(_))
 
-  property("Valid Action taken not 2,3,4,5,7 or 8") {
-    forAll(larGen, validActionTakenGen) { (lar, x) =>
+  property("Valid when Action taken not 2,3,4,5,7 or 8") {
+    forAll(larGen, irrelevantActionTakenGen) { (lar, x) =>
       val validLar = lar.copy(actionTakenType = x)
       validLar.mustPass
     }
   }
 
-  property("Valid if HOEPA status is 2") {
+  property("Valid when HOEPA status is 2") {
     forAll(larGen) { lar =>
       val validLar = lar.copy(hoepaStatus = 2)
       validLar.mustPass
     }
   }
 
-  val badActionTakenGen: Gen[Int] = Gen.oneOf(badActionTakenList)
+  val actionTakenGen: Gen[Int] = Gen.oneOf(relevantActionTakenList)
```

Comment:
Sure, that would be an appropriate use of `relevant`. Not sure why I didn't use it here--maybe because I thought it wasn't necessary, and that this Generator's purpose was clear enough.